Makefile

```diff
@@ -1,14 +1,40 @@
 UNAME := $(shell uname)
-PLAYCPPSDK=v0.0.6-alpha
-MACSRC=https://github.com/cronos-labs/play-cpp-sdk/releases/download/$(PLAYCPPSDK)/play_cpp_sdk_Darwin_x86_64.tar.gz
-MACCHEKSUMSRC=https://github.com/cronos-labs/play-cpp-sdk/releases/download/$(PLAYCPPSDK)/checksums-Darwin_x86_64.txt
-WINDOWSSRC=https://github.com/cronos-labs/play-cpp-sdk/releases/download/$(PLAYCPPSDK)/play_cpp_sdk_Windows_x86_64.zip
-WINDOWSCHECKSUMSRC=https://github.com/cronos-labs/play-cpp-sdk/releases/download/$(PLAYCPPSDK)/checksums-Windows_x86_64.txt
-LINUXSRC=https://github.com/cronos-labs/play-cpp-sdk/releases/download/$(PLAYCPPSDK)/play_cpp_sdk_libc++_Linux_x86_64.tar.gz
-LINUXCHECKSUMSRC=https://github.com/cronos-labs/play-cpp-sdk/releases/download/$(PLAYCPPSDK)/checksums-libc++_Linux_x86_64.txt
+PWD = $(shell pwd)
 
+# Set the play cpp sdk version
+PLAYCPPSDK=v0.0.7-alpha
+# Set NDK versions to be downloaded
+NDK_VERSION=21.4.7075529
+# Set names of play cpp sdk library files
+MAC_FILE=play_cpp_sdk_Darwin_x86_64.tar.gz
+WINDOWS_FILE=play_cpp_sdk_Windows_x86_64.zip
+LINUX_FILE=play_cpp_sdk_libc++_Linux_x86_64.tar.gz
+ARM64_V8A_FILE=play_cpp_sdk_aarch64-linux-android-$(NDK_VERSION).tar.gz
+ARMEABI_V7A_FILE=play_cpp_sdk_armv7-linux-androideabi-$(NDK_VERSION).tar.gz
+X86_64_FILE=play_cpp_sdk_x86_64-linux-android-$(NDK_VERSION).tar.gz
```

Comment:
maybe no need to support x86 if it is not supported in UE5 for Android?

Reply:
oh right, it should be x64.

Reply:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/92778185/186361026-081e3f1b-e9a8-47a0-9637-78aac7f6b4c0.png">

Reply:
x86_64 is disabled by default, to build it, we need to compile unreal from source 
<img width="389" alt="image" src="https://user-images.githubusercontent.com/92778185/186361278-831cd49d-e6b9-43fe-8c8c-01f897b0e301.png">

Reply:
This is screenshot from 4.27, it seems unreal 4.27 also does not support i686(x86).